playground/mocks/config/responseConfig.js

```diff
@@ -190,7 +190,8 @@ const idx = {
     'error-new-signup-email-exists'
     // 'authenticator-enroll-email',
     // 'authenticator-enroll-email-emailmagiclink-true',
-    //'authenticator-enroll-email-emailmagiclink-false'
+    // 'authenticator-enroll-email-emailmagiclink-false',
+    // 'authenticator-pre-enroll-email'
```

Comment:
Should `authenticator-pre-enroll-email-emailmagiclink-true` be added here too?

Reply:
no, because `authenticator-pre-enroll-email-emailmagiclink-true` is actually the remediation for the next screen after proceeding with the "send me an email" button, but it can be added to `/idp/idx/credential/enroll`, good call